### PR TITLE
timeline plugin: add offset parameter

### PR DESCRIPTION
**Pull request by moritzvieli**
### Short description of changes:
Sometimes it's required to adjust the timeline to the audio. For example, the timeline displays musical bars with a specific time signature. However, the audio does not start at 0 seconds, but maybe at 0.3 seconds. This new parameters allows passing an offset in seconds to the timeline plugin to adjust the timeline to the audio. The value may also be negative.

### Breaking in the external API:
None

### Breaking changes in the internal API:
None

### Todos/Notes:
What about documentation? If you can point me to a direction, I can check it. 🙂

### Related Issues and other PRs:
None

### CODE REVIEW

**Comment by moritzvieli** [comment]
@thijstriemstra done. Is it still missing somewhere?

**Comment by thijstriemstra** [comment]
@moritzvieli yea probably here: https://github.com/katspaugh/wavesurfer.js/blob/master/example/timeline/index.html#L96

File: src/plugin/timeline.js [hunk 1]
```diff
@@ -376,7 +379,8 @@ export default class TimelinePlugin {
             this.params.secondaryLabelInterval
         );
 
-        let curPixel = 0;
+        let curPixel =
```
Comment (thijstriemstra) [inline src/plugin/timeline.js]: you can simply use `this.params.offset` here (like line 376).

Comment (moritzvieli) [reply src/plugin/timeline.js]: True, thanks! 👍

**Comment by moritzvieli** [comment]
> @moritzvieli yea probably here: https://github.com/katspaugh/wavesurfer.js/blob/master/example/timeline/index.html#L96

@thijstriemstra True, thanks! I have not found any additional places to add the new param. Is it OK now?

**Comment by kjakobson** [comment]
Hi @thijstriemstra, first of all thanks for all the work you guys do! When do you expect release with this PR?

**Comment by thijstriemstra** [comment]
> When do you expect release with this PR?

@kjakobson opened #1651 for that

**Pull request merged**